ksql-functional-tests/src/test/java/io/confluent/ksql/test/EndToEndEngineTestUtil.java

```diff
@@ -495,31 +498,19 @@ private static KsqlEngine getKsqlEngine(final ServiceContext serviceContext) {
 
   static void shouldBuildAndExecuteQuery(final TestCase testCase) {
 
-
-    try (final ServiceContext serviceContext = getServiceContext();
-        final KsqlEngine ksqlEngine = getKsqlEngine(serviceContext)
-    ) {
-      testCase.initializeTopics(
-          serviceContext.getTopicClient(),
-          fakeKafkaService,
-          serviceContext.getSchemaRegistryClient());
-
-      final TopologyTestDriverContainer topologyTestDriverContainer =
-          buildStreamsTopologyTestDriverContainer(
```

Comment:
can we remove this method? I think this is the only usage of it

Reply:
Yes, removed it!